examples/typescript/swap.ts

```diff
@@ -0,0 +1,275 @@
+/**
+ * Example to demonstrate creating and adding to liquidity pools, swapping between two fungible asset.
+ *
+ * Steps:
+ * 1. Create two accounts (Alice and Bob)
+ * 2. Fund Alice and Bob
+ * 3. Create a FA with Alice as the owner
+ * 4. Create another FA with Bob as the owner
+ * 5. Create a liquidity pool with Alice's FA and Bob's FA
+ * 6. Swap between Alice's FA and Bob's FA
+ */
+
+import {
+  Account,
+  AccountAddress,
+  Aptos,
+  Bool,
+  Ed25519PrivateKey,
+  Hex,
+  HexInput,
+  MoveObject,
+  MoveString,
+  U64,
+  ViewRequestData,
+} from "aptos";
+
+const readline = require("readline").createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const getOptimalLpAmount = async (
+  aptos: Aptos,
+  swap: AccountAddress,
+  token1Addr: HexInput,
+  token2Addr: HexInput,
+): Promise<void> => {
+  const payload: ViewRequestData = {
+    function: `0x${swap.toStringWithoutPrefix()}::router::optimal_liquidity_amounts`,
+    arguments: [
```

Comment:
it was renamed to `functionArguments`, make sure you are working against the latest local sdk

Reply:
Why not arguments to make it shorter? 😜

Reply:
user feedback

Reply:
@movekevin it was because `arguments` is a reserved keyword in `strict` mode, so you couldn't easily deconstruct an args struct without having to rename:

```typescript
const { ..., arguments: functionArguments, ... } = args;
// use functionArguments...
``` 

if it leads to issues down the road we can change it back :p